docs/dev/how-to/send-transaction-l1-l2.md

```diff
@@ -0,0 +1,334 @@
+# Send an L1 to L2 transaction
+
+The [zkSync Era smart contracts](https://github.com/matter-labs/zksync-2-contracts/tree/main/ethereum/contracts/zksync) allow the sender to request transactions on Ethereum L1 and pass data to zkSync Era L2.
+
+## Common use cases
+
+Along with zkSync Era's built-in censorship resistance that requires multi-layer interoperability, there are some common use cases that need L1 to L2 transaction functionality, such as:
+
+- Custom bridges.
+- Multi-layer governing smart contracts.
+
+## Step-by-step
+
+1. Import the zkSync Era library or contract containing the required functionality. 
+
+    The import gives access to the [`IZkSync.sol`](https://github.com/matter-labs/v2-testnet-contracts/blob/b8449bf9c819098cc8bfee0549ff5094456be51d/l1/contracts/zksync/interfaces/IZkSync.sol#L4) inherited interfaces that include the gas estimation functionality.
+
+    Import the contracts with yarn (recommended), or [download the contracts](https://github.com/matter-labs/v2-testnet-contracts) from the repo.
+
+    ::: code-tabs
+    @tab yarn
+    ```yarn
+    yarn add -D @matterlabs/zksync-contracts
+    ```
+    :::
+
+2. Get the current L1 gas price with Ethereum JSON-RPC method [`eth_gasPrice`](https://ethereum.github.io/execution-apis/api-documentation/).
+
+    ::: code-tabs
+    @tab TypeScript
+    ```ts
+    // Method available on a zkSync Era JS SDK Provider object
+    override async getGasPrice(token?: Address): Promise<BigNumber> {
```

Comment:
The zkSync Era JS SDK should not be used to obtain the gas price on L1. The gas price on L1 should be obtained via ethers.js SDK

Reply:
updated